frontend/src/components/message.tsx

```diff
@@ -59,12 +61,12 @@ export const MessageComponent = ({
         <img src={icon} className={styles.iconStyle} />
         <p className={styles.messageStyle}>{content}</p>
       </div>
-      {role == Role.Bot && false && (
+      {sidePanelContent && (
         <div className={styles.selectMessage}>
           <Button
             isOutline
             isPressed={message === selectedMessage}
-            text="Select message"
+            text="View Report"
```

Comment:
Can we also pass `buttonText` in? This way Mike will be able to pass in `View Data` or something like this